src/baseline_rule.py

```diff
@@ -88,6 +107,28 @@ def matches_target(self, labels):
         """
         return BaselineRule._matches_selectors(labels, self.target)
 
+    def matches_source_namespace(self, labels):
+        """
+        Check whether the given set of labels match the rule source's namespace
```

Comment:
```suggestion
        Check whether the given set of labels match the rule source's namespace
        NOTE: if no labels are provided, this is interpreted as "no namespace context", and the rule matches
```
Also add this comment in `matches_target_namespace`